x/dispensation/handler_test.go

```diff
@@ -74,6 +75,196 @@ func TestNewHandler_CreateDistribution_MultipleTypes(t *testing.T) {
 	require.NotNil(t, res)
 }
 
+func TestNewHandler_CreateDistribution_PayRewardsInAnyToken_HappyCase(t *testing.T) {
+	app, ctx := test.CreateTestApp(false)
+	keeper := app.DispensationKeeper
+	handler := dispensation.NewHandler(keeper)
+	recipients := 3
+	outputList := test.CreatOutputList(recipients, "10")
+	runner := sdk.AccAddress(ed25519.GenPrivKey().PubKey().Address())
+	distributor := sdk.AccAddress(ed25519.GenPrivKey().PubKey().Address())
+	totalCoins, err := dispensationUtils.TotalOutput(outputList)
+	assert.NoError(t, err)
+	totalCoins = totalCoins.Add(totalCoins...)
+	err = sifapp.AddCoinsToAccount(types.ModuleName, app.BankKeeper, ctx, distributor, totalCoins)
+	assert.NoError(t, err)
+	// Airdrop distribution type
+	msgAirdrop := types.NewMsgCreateDistribution(distributor, types.DistributionType_DISTRIBUTION_TYPE_AIRDROP, outputList, runner.String())
+	res, err := handler(ctx, &msgAirdrop)
+	require.NoError(t, err)
+	require.NotNil(t, res)
+	res, err = handler(ctx, &msgAirdrop)
```

Comment:
why is this being created twice?

Reply:
Second handler will fail and we catch that at line 98 and line 99.